src/NetworkTest/testQuality/helpers/config.ts

```diff
@@ -38,34 +38,28 @@ const config: QualityTestConfig = {
   qualityThresholds: {
     video: [
       {
-        bps: 1000000,
-        plr: 0.005,
-        recommendedSetting: '1280x720 @ 30FPS',
+        bps: 4000000,
```

Comment:
We risk being too conservative by using these values, since the browser will actually start enoding at FHD (for vp8) with much less available bitrate than the target bitrate, which is codec independent. See https://jira.vonage.com/browse/VIDMR-73

Reply:
Hey @andresbersier, we are currently using the ongoing available bitrate (I have renamed the variable). Based on my observations, it seems that whenever the outgoingAvailableBitrate reaches the target bitrate, the browser initiates a resolution change or adjusts the codec, regardless of the specific bitrate we are sending. I have noticed clear resolution changes during these values in my tests.

Reply:
@andresbersier as discussed, added a threshold ratio of 0.75